### DISCUSSION TITLE
现版本网页播放器Artplayer不支持内封字幕，请求更新

### DISCUSSION THREAD

**Discussion by dengletail** [AlistGo/alist#1134] [Ideas]
现在alist网页端播放器Artplayer的版本是4.3.1，我发现并不支持内封字幕，只能外挂，没有调字幕的选项。但我看了下现在最新版的artplayer是4.4.2版本，已经支持内封字幕，并且还能调整字幕。请求更新Artplayer的版本到最新版，谢谢~

**Comment by simpleandstupid** [comment]
支持

**Comment by AspirantH** [comment]
有同样的需求,请求支持!

**Comment by Yunmengxian** [comment]
支持

**Comment by Nagileo** [comment]
支持

**Comment by ffinly** [comment]
支持，目前封装的版本已经是4.4.3了，artplayer官方最新的是4.5.3，但是从示例代码来看，字幕也不是内封的，是调用的外部字幕文件
<img width="607" alt="image" src="https://user-images.githubusercontent.com/29793346/185776521-4401ab55-7815-4bf8-94a1-0a6d52a94ed0.png">

**Comment by Tonge-Yang** [comment]
怎么调用外部字幕啊，我同名的.ass字幕没被调用求指导 http://gewuzhizhi.tpddns.cn:1880/Videos/%E7%81%B0%E8%89%B2%E4%B8%89%E9%83%A8%E6%9B%B2_%E6%89%B3%E6%9C%BA/%E7%81%B0%E8%89%B2%E7%9A%84%E6%9E%9C%E5%AE%9E/[Seed-Raws]%20Grisaia%20no%20Kajitsu%20-%2002%20(BD%201280x720%20AVC%20AAC).mp4
更新，已经好了，可能是卡的原因

**Comment by MkaPoi** [comment]
支持

**Comment by chai-mi** [comment]
支持

**Comment by BraveSail** [comment]
artplayer不支持内封字幕，官网的演示也是外挂的srt

**Reply by yibaogang** [reply]
挂在不了ASS字幕吧？  我同名的ASS字幕 根本不显示，但是播放器github页面 确实有说是支持ass
SRT字幕 双语效果真心的不好，还是很期待能支持ASS字幕的

**Comment by kimycai** [comment]
但是ass字幕无法调整字体大小，不知道srt行不行，试试看再回
